settings/settings-cpt.php

```diff
@@ -131,9 +138,16 @@ protected function form() {
 						printf(
 							'<li><label><input name="taxonomies[%s]" type="checkbox" value="1" %s %s/> %s</label></li>',
 							esc_attr( $taxonomy ),
-							checked( in_array( $taxonomy, $this->options['taxonomies'] ) || $disabled, true, false ),
+							checked( $disabled || in_array( $taxonomy, $this->options['taxonomies'] ), true, false ),
```

Comment:
We're missing the strict comparison here too:
```suggestion
							checked( $disabled || in_array( $taxonomy, $this->options['taxonomies'], true ), true, false ),
```